js/src/components/google-combo-account-card/connected-google-combo-account-card.js

```diff
@@ -110,24 +115,29 @@ const ConnectedGoogleComboAccountCard = () => {
 	};
 
 	return (
-		<div className="gla-google-combo-account-cards">
-			<AccountCard
-				appearance={ APPEARANCE.GOOGLE }
-				className="gla-google-combo-account-card gla-google-combo-account-card--connected"
-				description={
-					<AccountCreationDescription
-						isCreatingBothAccounts={ isCreatingBothAccounts }
-						isCreatingAdsAccount={ isCreatingAdsAccount }
-						isCreatingMCAccount={ isCreatingMCAccount }
-						accountsCreated={ accountsCreated }
-					/>
-				}
-				helper={ getHelper() }
-				indicator={ getIndicator() }
+		<AccountCard
+			appearance={ APPEARANCE.GOOGLE }
+			className="gla-google-combo-account-card gla-google-combo-account-card--connected"
+			description={
+				<AccountCreationDescription
+					hasExistingMCAccounts={ hasExistingMCAccounts }
+					isCreatingBothAccounts={ isCreatingBothAccounts }
+					isCreatingAdsAccount={ isCreatingAdsAccount }
+					isCreatingMCAccount={
+						isCreatingMCAccount || isManuallyCreatingMCAccount
+					}
+					accountsCreated={ accountsCreated }
+					isGoogleMCAccountConnected={ isGoogleMCAccountConnected }
+					isGoogleAdsAccountConnected={ isGoogleAdsAccountConnected }
+				/>
+			}
+			helper={ getHelper() }
+			indicator={ getIndicator() }
+		>
+			<ConnectMC
```

Comment:
I think it's better to have a set of cards that are visually container together where you have a list of `AccountCard` components where the first one is the Google card where you have all the info displayed and the other cards are for MC and Ads. What do you think?

Reply:
While I think it would be great to compose the entire `GoogleComboAccountCard` so that it always rendered a single `AccountCard` component with the MC and Ads subsections returning `<Section.Card.Body>` components, this would require a bigger refactor of the existing `GoogleMCAccountCard` and possibly the `GoogleAdsAccountCard` components in order to share components like the `ReclaimUrlCard` and `SwitchUrlCard` that are repurposed in this design, so this does feel like more of a grouping of `AccountCard` components rather than a single card.